pkg/littlefs/Makefile

```diff
@@ -1,15 +1,12 @@
 PKG_NAME=littlefs
 PKG_URL=https://github.com/geky/littlefs.git
-PKG_VERSION=be22d3449f23a24e7462114349ce04d751e42437
+PKG_VERSION=v1.3
 PKG_BUILDDIR ?= $(PKGDIRBASE)/$(PKG_NAME)
```

Comment:
Maybe the PKG_BUILDDIR line could also be removed. I think it's set automatically.

Reply:
Amended directly